Core/clim-basic/geometry/region-set-composition.lisp

```diff
@@ -0,0 +1,192 @@
+(in-package #:climi)
+
+
+;;; utilities
+(defun rectangle-set->polygon-union (rs)
+  (let ((res nil))
+    (map-over-region-set-regions (lambda (r) (push r res)) rs)
+    (make-instance 'standard-region-union :regions res)))
+
+
+;; REGION-UNION
+
+#+ (or) ;; defined in region-composition.lisp
```

Comment:
Can this be removed? Since the method has no region set specializer, I wouldn't expect it to be defined in this file.

Reply:
fixed